conda/recipes/libcuopt/recipe.yaml

```
@@ -149,6 +152,8 @@ outputs:
         - rapids-logger =0.1
         - librmm =${{ dep_minor_version }}
         - cuda-cudart-dev
+        - cuda-driver-dev
+        - libcudss-dev
         - libcublas
```

Alphabetize this whole section?
```suggestion
        - libcublas
        - libcudss-dev
```